icons/banknote-x.json

```
@@ -0,0 +1,21 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "AnnaSasDev",
+    "joffx"
```

Please also add the original contributors from `banknote`, the icon this was based upon.

```suggestion
    "AnnaSasDev",
    "joffx",
    "ericfennis",
    "danielbayley"
```